modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -146,6 +146,17 @@ The following permissions are not required to create the private {product-title}
 * `Microsoft.Storage/storageAccounts/write`
 ====
 
+.Optional permissions for creating an image registry storage account private endpoint
```

Comment:
Maybe  replace Optional with Required

Reply:
+1
Making the storage account private is optional, but when users choose to do so then these permissions are required.

Reply:
@wewang58 @flavianmissi Set to "Required"

Reply:
@stevsmit  Sorry, after discussed with @jinyunma  it should Optional, it means it's just a optional permission only you when you create an image registry storage account private endpoint, can you revert it again, thanks